### DISCUSSION TITLE
{   "message": "spawn /usr/sbin/sendmail ENOENT",   "extensions": {     "code": "INTERNAL_SERVER_ERROR"   } }

### DISCUSSION THREAD

**Discussion by alexandrestevens** [directus/directus#11779] [Archive]
### Describe the Bug

I installed Directus 1-click on Digitalocean without any problem yesterday and configured a subdomain. Everything was working normally. Today, I tried to configure a second server on another subdomain and now I can't invite people. 

### To Reproduce

Sending emails does not work but user is created. I get the same error on yesterday's and today's server when I try to add a user.

Any idea ?

### Errors Shown

{   "message": "spawn /usr/sbin/sendmail ENOENT",   "extensions": {     "code": "INTERNAL_SERVER_ERROR"   } }

### What version of Directus are you using?

9.5.2

### What version of Node.js are you using?

14.15.4

### What database are you using?

PostgreSQL 12

### What browser are you using?

Chrome

### What operating system are you using?

Linux 5.4.0-62-generic

### How are you deploying Directus?

1-Click DigitalOcean

**Comment by rijkvanzanten** [comment]
Looks like you don't have Sendmail installed on the droplet. You'll either have to install and configure `sendmail`, or use an email service like [Sendgrid](https://sendgrid.com) or [AWS SES](https://aws.amazon.com/ses/) to send emails.

(I also think DigitalOcean might block outgoing emails by default for security reasons, but I'm not 100% sure)

**Comment by alexandrestevens** [comment]
OKé bedankt Rijk. Strange it doesn't come with 1-click install on their side. Thanx for fast reply.

**Comment by sawyerclick** [comment]
I am running into this issue as well. I've ran `apt-get install sendmail` but after install emails are delayed and sent straight to spam. What set-up worked for you?

**Reply by alexandrestevens** [reply]
I did not manage to install sendmail yet... sorry

**Comment by sawyerclick** [comment]
I've gotten a set-up to work by doing the following:

1. Install sendmail: `apt-get install sendmail`
2. change the first line of /etc/hosts/ from `127.0.0.1 localhost hostname` to read as `127.0.0.1 example.com hostname`
3. Reload sendmail with the new configuration: `sendmailconfig`. Hit Y/enter to all popups.
4. In /var/direcuts/.env update PUBLIC_URL to match your public domain (ex: "https://www.example.com") and update EMAIL_FROM to the desired no-reply address.
5. Add SSL/HTTPS via the `npx directus https` command. IMPORTANT: Have your A and AAAA records already set up in your DNS records. You will also need to add these in Digital Ocean for the Droplet.
6. Restart the Directus app by running the command `npx directus pm2 restart directus --update-env`
